integration-tests/src/test/java/com/ca/mfaas/discoveryservice/staticdef/StaticClientEndpointsTest.java

```diff
@@ -34,4 +35,14 @@ public void shouldSameResponseAsDynamicService() throws Exception {
         assertEquals(response.getStatusLine().getStatusCode(), staticResponse.getStatusLine().getStatusCode());
         assertEquals(hello, staticHello);
     }
+
+    /**
+     * Test service with mixed case service ID
+     * Depends on service StaticClient3 to be defined in staticclient.yml
+     * @throws Exception
+     */
+    @Test
+    public void shouldMixedCaseServiceIdWork() throws Exception {
+        final HttpResponse staticResponse = HttpRequestUtils.getResponse(MIXED_CASE_GREET, SC_OK);
```

Comment:
Please, don't need to create a variable, if you don't use it. Thanks.

Reply:
Thank you for the comments, I've modified the code in latest commit